ocs_ci/deployment/deployment.py

```diff
@@ -225,6 +228,68 @@ def create_catalog_source(self):
         # Wait for catalog source is ready
         catalog_source.wait_for_state("READY")
 
+    def create_stage_operator_source(self):
+        """
+        This prepare operator source for OCS deployment from stage.
+        """
+        logger.info("Adding Stage Secret")
+        # generate quay token
+        credentials = {
+            "user": {
+                "username": config.DEPLOYMENT.get("stage_quay_username"),
+                "password": config.DEPLOYMENT.get("stage_quay_password"),
+            }
+        }
+        token = requests.post(
+            url='https://quay.io/cnr/api/v1/users/login',
+            data=json.dumps(credentials),
+            headers={'Content-Type': 'application/json'},
+        ).json()['token']
+        stage_ns = config.DEPLOYMENT.get("stage_namespace")
```

Comment:
What about to have some default stage_namespace defined and used here:
`config.DEPLOYMENT.get("stage_namespace", defaults.STAGE_NAMESPCE)`
something like this. Or if we don't want to have default stage namespace I would also directly access key without get method for the reason mentioned above.

Reply:
Since the namespace is tightly connected to the used credentials (or vice versa), I think it is better to make the `stage_namespace` also mandatory. Fixed by 6ab36d8.